Packs/PaloAltoNetworks_SecurityAdvisories/Scripts/CheckPanosVersionAffected/CheckPanosVersionAffected.py

```diff
@@ -50,6 +50,12 @@ def return_result_dataclass(result: list[Advisory]):
         summary_list = [vars(x) for x in result]
         title = result[0]._title
         output_prefix = result[0]._output_prefix
+    else:
```

Comment:
for next time, I'd simply remove the `if` condition since `result` is guaranteed to be a list. We will never be in the else block